src/store.js

```diff
@@ -0,0 +1,19 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+export const fetchMessage = createAsyncThunk('message/fetchMessage', async () => {
+  const response = await fetch('http://localhost:3000/api/v1/messages/random');
+  const data = await response.json();
+  return data.greeting;
+});
+const messageSlice = createSlice({
+  name: 'message',
+  initialState: { message: '' },
+  reducers: {},
+  extraReducers: (builder) => {
+    builder.addCase(fetchMessage.fulfilled,
+      (state, action) => ({ ...state, message: action.payload }
+      ));
+  },
+});
```

Comment:
- [ ] You didn't create a store for your reducer. you need to create a store that holds the whole state tree of your application. It is through the store that components can connect to the application's state and change the state through the dispatch of actions. 👍